operations/iuf/stages/deliver_product.md

```diff
@@ -58,6 +61,14 @@ The following table describes upload behavior when the artifact being uploaded a
 
 (`ncn-m001#`) Execute the `deliver-product` stage for activity `admin-230127`.
 
+### Option 1: Install or upgrade without SLURM or PBS Products
+
 ```bash
 iuf -a admin-230127 run -r deliver-product
 ```
+
+### Option 2: Install or upgrade along with SLURM or PBS Products
+
+```bash
+iuf -a admin-230127 run -sv /etc/cray/upgrade/csm/admin/site_vars.yaml -bpcd /etc/cray/upgrade/csm/admin -r deliver-product
+```
```

Comment:
In order to enable a workload manager, admin also need to include the site variables file with the `deploy-product` stage. I think we need to include these changes in the [`deploy-product` topic](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/iuf/stages/deploy_product.md) as well.

- We should add line 23 above to the `deploy-product` table.
- We should add similar options to the `deploy-product` "examples" heading. Note that the `-sv` argument needs to be added, but the WLM does not need to be selected in the site variable file again (since it was already selected here for the `deliver-product` stage).